> @sobolkz If you flash a factory firmware ALWAYS check `DoNotChgBin` ! ![185151427-3cf65e64-711c-49ad-a6e2-f35b3555fa70](https://user-images.githubusercontent.com/24528715/185155055-62aa470c-2cdc-41c9-8501-46548732f042.jpg)

Fixed:
![image](https://user-images.githubusercontent.com/55306010/185185829-92f610bb-0632-43d0-a51f-5e26004512d7.png)

But no changes with module adding:

> ets Jul 29 2019 12:21:46
> 
> rst:0x1 (POWERON_RESET),boot:0x17 (SPI_FAST_FLASH_BOOT)
> configsip: 0, SPIWP:0xee
> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
> mode:DOUT, clock div:2
> load:0x3fff0030,len:184
> load:0x40078000,len:12616
> load:0x40080400,len:2916
> entry 0x400805c4
> ▒U42 : Info   :
> 
> INIT : Booting version: HEAD_3a9783d (ESP32 SDK v4.4.3)
> 43 : Info   : INIT : Free RAM:267484
> 44 : Info   : INIT : Cold Boot - Restart Reason: CPU0: Vb./components/esp_littlefs/src/littlefs/lfs.c:1229:error: Corrupted dir pair at {0x0, 0x1}
> at power on reset CPU1: for APP CPU, reseted by PRO CPU
> 45 : Info   : FS   : Mounting...
> FS   : Mount failed
> 54 : Error  : FS   : Mount failed
> RESET: Reset./components/esp_littlefs/src/littlefs/lfs.c:1229:error: Corrupted dir pair at {0x0, 0x1}
> ting factory defaults... using default settings
> RESET: Warm boot, reset count: 0
> RESET: formatting...
> RESET: formatting done...
> 30422 : Info   : SaveToFile: free stack: 7416
> 32449 : Info   : FILE : Saved config.dat offset: 8192 size: 820
> 32450 : Info   : SaveToFile: free stack after: 7416
> 32451 : Info   : SaveToFile: free stack: 7448
> 34630 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 34631 : Info   : SaveToFile: free stack after: 7448
> 34632 : Info   : SaveToFile: free stack: 7256
> 34745 : Info   : FILE : Saved security.dat offset: 0 size: 593
> 34746 : Info   : SaveToFile: free stack after: 7256
> 34759 : Info   : SaveToFile: free stack: 7000
> 34844 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 34845 : Info   : SaveToFile: free stack after: 7000
> RESET: Successful, rebooting. (you might need to press the reset button if you've just flashed the firmware)
> 36023 : Info   : WIFI : Set WiFi to OFFets Jul 29 2019 12:21:46
> 
> rst:0xc (SW_CPU_RESET),boot:0x17 (SPI_FAST_FLASH_BOOT)
> configsip: 0, SPIWP:0xee
> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
> mode:DOUT, clock div:2
> load:0x3fff0030,len:184
> load:0x40078000,len:12616
> load:0x40080400,len:2916
> entry 0x400805c4
> ▒U47 : Info   :
> 
> INIT : Booting version: HEAD_3a9783d (ESP32 SDK v4.4.3)
> 48 : Info   : INIT : Free RAM:267164
> 50 : Info   : INIT : Soft Reboot #1 Last Action before Reboot: Intended Reboot: ResetFactory Last systime: 36 - Restart Reason: CPU0: Software reset CPU CPU1: Software reset CPU
> 51 : Info   : FS   : Mounting...
> 91 : Info   : FS   : Mount successful, used 151552 bytes of 8318976
> 183 : Info   : CRC  : SecuritySettings CRC   ...OK
> 206 : Info   : INIT : I2C
> 208 : Info   : INIT : SPI not enabled
> 209 : Info   : Time set to 36.000 Time adjusted by 35791.00 msec. Wander: 0.000 msec/second Source: RTC at boot
> 211 : Info   : Current Time Zone: STD time start: 1970-10-25 03:00:00 offset: 0 min
> 212 : Info   : Local time: 1970-01-01 00:00:36
> 214 : Info   : WiFi : Start network scan all channels
> 315 : Info   : WiFi : STA Started
> 7330 : Info   : WiFi : Scan finished, found: 18
> 7334 : Error  : WIFI : No valid wifi settings
> 7436 : Info   : WIFI : Set WiFi to AP+STA
> 7439 : Info   : WiFi : AP Started
> 8001 : Info   : WiFi : AP Stopped
> 8002 : Info   : WiFi : AP Started
> 8003 : Info   : WIFI : AP Mode ssid will be MAX-ESP32 with address 192.168.4.1
> 8005 : Info   : WIFI : Arduino wifi status: WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 8110 : Info   : WIFI : Disconnected! Reason: '(8) Assoc leave' Connected for 8008 ms
> 8125 : Error  : WiFi : Scan not allowed, unprocessed WiFi events:  disconn
> 8227 : Info   : WIFI : Set WiFi to OFF
> 8451 : Info   : WiFi : Start network scan all channels
> 8463 : Info   : WiFi : STA Started
> 15276 : Info   : WiFi : Scan finished, found: 13
> 15383 : Info   : WIFI : Set WiFi to AP+STA
> 15947 : Info   : WIFI : AP Mode ssid will be MAX-ESP32 with address 192.168.4.1
> 16049 : Info   : WIFI : Disconnected! Reason: '(8) Assoc leave' Connected for 15 s
> 16065 : Error  : WiFi : Scan not allowed, unprocessed WiFi events:  disconn
> 16166 : Info   : WIFI : Set WiFi to OFF
> 16390 : Info   : WIFI : Arduino wifi status: WL_NO_SHIELD 255 ESPeasy internal wifi status: DISCONNECTED
> 16494 : Info   : WIFI : Set WiFi to AP
> 16505 : Info   : WiFi : AP Started
> 17067 : Info   : WiFi : AP Stopped
> 17068 : Info   : WiFi : AP Started
> 17069 : Info   : WIFI : AP Mode ssid will be MAX-ESP32 with address 192.168.4.1
> 17105 : Info   : INIT : Free RAM:214452
> 17633 : Info   : INFO : Plugins: 126 [Normal] [MAX ESP32] (ESP32 SDK v4.4.3)
> 17637 : Info   : WIFI : Arduino wifi status: WL_NO_SHIELD 255 ESPeasy internal wifi status: DISCONNECTED
> 17643 : Info   : Webserver: start
> 17649 : Info   : OTA  : Arduino OTA enabled on port 3232
> 18996 : Info   : WD   : Uptime 0 ConnectFailures 0 FreeMem 194696 WiFiStatus WL_NO_SHIELD 255 ESPeasy internal wifi status: DISCONNECTED
> 24088 : Info   : AP Mode: Client connected: 54:27:1E:06:27:B4 Connected devices: 1
> 24105 : Error  : UNKNOWN WIFI/ETH EVENT: 14
> 46135 : Info   : SaveToFile: free stack: 7208
> 48331 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 48332 : Info   : SaveToFile: free stack after: 7208
> 48333 : Info   : SaveToFile: free stack: 7016
> 48443 : Info   : FILE : Saved security.dat offset: 0 size: 593
> 48444 : Info   : SaveToFile: free stack after: 7016
> 48462 : Info   : SaveToFile: free stack: 6760
> 48557 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 48559 : Info   : SaveToFile: free stack after: 6760
> 48560 : Info   : WIFI : Credentials Changed, retry connection. SSID: access_point8
> 48562 : Info   : WiFi : Best AP candidate: access_point8 88:DC:96:26:7F:0A Ch:9 (-59dBm) WPA2/PSK
> 48563 : Info   : WIFI : Connecting access_point8 88:DC:96:26:7F:0A Ch:9 (-59dBm) WPA2/PSK attempt #0
> 48664 : Info   : WIFI : Set WiFi to AP+STA
> 48668 : Info   : WiFi : STA Started
> 48775 : Info   : WIFI : Arduino wifi status: WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 49014 : Info   : WD   : Uptime 1 ConnectFailures 0 FreeMem 186004 WiFiStatus WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 49671 : Info   : WIFI : Connected! AP: access_point8 (88:DC:96:26:7F:0A) Ch: 9 Duration: 1101 ms
> 49696 : Info   : WIFI : DHCP IP: 192.168.3.38 (MAX-ESP32) GW: 192.168.3.1 SN: 255.255.255.0   duration: 29 ms
> 49716 : Info   : SaveToFile: free stack: 7096
> 49811 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 49812 : Info   : SaveToFile: free stack after: 7096
> 49813 : Info   : WIFI : Arduino wifi status: WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP
> 49813 : Info   : WIFI : Starting mDNS...
> 49817 : Info   : WIFI : mDNS started, with name: MAX-ESP32.local
> 49830 : Info   : UDP : Start listening on port 8266
> 49832 : Info   : firstLoopConnectionsEstablished
> 51890 : Info   : AP Mode: Client disconnected: 54:27:1E:06:27:B4 Connected devices: 1
> 56731 : Error  : UNKNOWN WIFI/ETH EVENT: 14
> 56734 : Info   : AP Mode: Client connected: 54:27:1E:06:27:B4 Connected devices: 1
> 78983 : Info   : WD   : Uptime 1 ConnectFailures 0 FreeMem 181424 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 108983 : Info   : WD   : Uptime 2 ConnectFailures 0 FreeMem 180724 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 138983 : Info   : WD   : Uptime 2 ConnectFailures 0 FreeMem 181748 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 144936 : Info   :  Webserver 35 Arguments (First 20): 0: 'userules' length: 2 1: 'enable_rules_cache' length: 2 2: 'usentp' length: 2 3: 'ntphost' length: 11 4: 'exttimesource' length: 1 5: 'dststartweek' length: 1 6: 'dststartdow' length: 1 7: 'dststartmonth' length: 1 8: 'dststarthour' length: 1 9: 'dstendweek' length: 1 10: 'dstenddow' length: 1 11: 'dstendmonth' length: 2 12: 'dstendhour' length: 1 13: 'timezone' length: 3 14: 'latitude' length: 8 15: 'longitude' length: 8 16: 'syslogip' length: 0 17: 'syslogport' length: 3 18: 'syslog_log_level' length: 1 19: 'syslogfacility' length: 1
> 144947 : Info   : SaveToFile: free stack: 7240
> 147143 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 147145 : Info   : SaveToFile: free stack after: 7240
> 147145 : Debug dev : Write settings: ExtendedControllerCredentials index: 0
> 147165 : Info   : SaveToFile: free stack: 6792
> 147263 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 147265 : Info   : SaveToFile: free stack after: 6792
> 147268 : Debug More : NTP  : NTP host 192.168.3.1 (192.168.3.1) queried
> 147282 : Info   : NTP  : NTP replied: delay 11 mSec Accuracy increased by 0.699 seconds
> 147285 : Info   : Time set to 1660751388.699
> 147286 : Info   : Current Time Zone: STD time start: 2022-10-30 03:00:00 offset: 360 min
> 147287 : Info   : Local time: 2022-08-17 21:49:48
> 147292 : Debug dev : sendcontent free: 177452 chunk size:1360
> 147307 : Debug dev : sendcontent free: 175288 chunk size:1360
> 147327 : Debug dev : sendcontent free: 176796 chunk size:1360
> 147339 : Debug dev : sendcontent free: 170020 chunk size:1360
> 147351 : Debug dev : sendcontent free: 165568 chunk size:1360
> 147366 : Debug dev : sendcontent free: 175524 chunk size:1360
> 147379 : Debug dev : sendcontent free: 176868 chunk size:1360
> 147391 : Debug dev : sendcontent free: 175236 chunk size:1360
> 147405 : Debug dev : sendcontent free: 176708 chunk size:1360
> 147425 : Debug dev : sendcontent free: 176764 chunk size:1360
> 147437 : Debug dev : sendcontent free: 171688 chunk size:217
> 147449 : Debug dev : sendcontent free: 173032 chunk size:0
> 147465 : Info   : EVENT: Time#Set
> 147667 : Debug  : Rules : Read 0 lines from rules1.txt
> 147678 : Debug  : EVENT: Time#Set Processing time:213 milliSeconds
> 147691 : Info   : EVENT: Clock#Time=Wed,21:49
> 147692 : Debug  : EVENT: Clock#Time=Wed,21:49 Processing time:1 milliSeconds
> 151511 : Debug More : UDP  : 24:6F:28:D1:9A:D8,192.168.3.3,1
> 155951 : Debug dev : sendcontent free: 176452 chunk size:1360
> 155964 : Debug dev : sendcontent free: 173100 chunk size:1360
> 155974 : Debug dev : sendcontent free: 171508 chunk size:1360
> 155983 : Debug dev : sendcontent free: 168524 chunk size:1360
> 155997 : Debug dev : sendcontent free: 177984 chunk size:1360
> 156014 : Debug dev : sendcontent free: 171068 chunk size:317
> 156036 : Debug dev : sendcontent free: 171460 chunk size:1360
> 156050 : Debug dev : sendcontent free: 176180 chunk size:1360
> 156063 : Debug dev : DEBUG: String size:9837
> 156066 : Debug dev : sendcontent free: 177824 chunk size:1344
> 156073 : Debug dev : sendcontent free: 173068 chunk size:0
> 157230 : Debug dev : sendcontent free: 176032 chunk size:29
> 157242 : Debug dev : sendcontent free: 174612 chunk size:0
> 157473 : Debug dev : sendcontent free: 175700 chunk size:1360
> 157486 : Debug dev : sendcontent free: 172336 chunk size:1360
> 157500 : Debug dev : sendcontent free: 175496 chunk size:1360
> 157512 : Debug dev : sendcontent free: 174016 chunk size:1360
> 157525 : Debug dev : sendcontent free: 175700 chunk size:1360
> 157538 : Info   :  Webserver 2 Arguments: 0: 'index' length: 1 1: 'page' length: 1
> 157545 : Debug dev : sendcontent free: 175508 chunk size:1360
> 157562 : Debug dev : sendcontent free: 175620 chunk size:1360
> 157579 : Debug dev : sendcontent free: 177204 chunk size:1360
> 157596 : Debug dev : sendcontent free: 175620 chunk size:1360
> 157613 : Debug dev : sendcontent free: 177204 chunk size:1360
> 157629 : Debug dev : sendcontent free: 175700 chunk size:1360
> 157643 : Debug dev : DEBUG: String size:14960
> 157646 : Debug dev : sendcontent free: 177200 chunk size:1071
> 157657 : Debug dev : sendcontent free: 172232 chunk size:0
> 158691 : Info   : EVENT: Clock#Time=Wed,21:50
> 158692 : Debug  : EVENT: Clock#Time=Wed,21:50 Processing time:1 milliSeconds
> 162760 : Debug dev : sendcontent free: 175524 chunk size:1360
> 162774 : Debug dev : sendcontent free: 173696 chunk size:1360
> 162786 : Debug dev : sendcontent free: 172212 chunk size:1360
> 162798 : Debug dev : sendcontent free: 173736 chunk size:1360
> 162811 : Debug dev : sendcontent free: 175516 chunk size:1360
> 162822 : Info   :  Webserver 5 Arguments: 0: 'index' length: 1 1: 'page' length: 1 2: 'TDNUM' length: 2 3: 'edit' length: 1 4: 'page' length: 1
> 162826 : Debug dev : Write settings: CustomTaskSettings index: 0
> 164446 : Debug dev : Write settings: TaskSettings index: 0
> 164448 : Info   : SaveToFile: free stack: 7096
> 166058 : Info   : FILE : Saved config.dat offset: 32768 size: 536
> 166059 : Info   : SaveToFile: free stack after: 7096
> 166061 : Info   : SaveToFile: free stack: 7176
> 168212 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 168213 : Info   : SaveToFile: free stack after: 7176
> 168214 : Debug dev : Write settings: ExtendedControllerCredentials index: 0
> 168225 : Info   : SaveToFile: free stack: 6728
> 168308 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 168310 : Info   : SaveToFile: free stack after: 6728
> 168311 : Debug dev : Read settings: TaskSettings index: 0
> 168354 : Debug dev : Read settings: TaskSettings index: 0
> 168375 : Debug dev : sendcontent free: 176796 chunk size:1149
> 168402 : Debug dev : sendcontent free: 175084 chunk size:1360
> 168420 : Debug dev : sendcontent free: 175108 chunk size:1360
> 168439 : Debug dev : sendcontent free: 176652 chunk size:1360
> 168452 : Debug dev : sendcontent free: 169924 chunk size:1360
> 168469 : Debug dev : sendcontent free: 176716 chunk size:1360
> 168481 : Debug dev : sendcontent free: 175068 chunk size:1360
> 168498 : Debug dev : sendcontent free: 176716 chunk size:1360
> 168520 : Debug dev : sendcontent free: 175044 chunk size:1360
> 168531 : Debug dev : DEBUG: String size:18829
> 168537 : Debug dev : sendcontent free: 176572 chunk size:766
> 168543 : Debug dev : sendcontent free: 170440 chunk size:0
> 168566 : Info   : EVENT: TaskExit#=1,0
> 168599 : Debug  : Rules : Read 0 lines from rules1.txt
> 168615 : Debug  : EVENT: TaskExit#=1,0 Processing time:49 milliSeconds
> 168983 : Debug dev : Memtrace
>                              0: lowest: 170440 PluginCall_task_1 (Unknown)-> 176724 PluginCall (9)-> 176828 PluginCall_task_1 (WEBFORM_LOAD)-> 176688 sendContentBlocking-> 176652 sendContentBlocking-> 169924 sendContentBlocking-> 176716 sendContentBlocking-> 175068 PluginCall (13)-> 176732 PluginCall_task_1 (Unknown)-> 176660 sendContentBlocking-> 176716 sendContentBlocking-> 175044 handle_devices-> 173268 sendWebPage-> 176620 sendContentBlocking-> 176572 sendContentBlocking-> 170440
> 168985 : Debug dev : 1: lowest: 166680 PluginCall (22)-> 177500 PluginCall (22)-> 179212 PluginCall (22)-> 179212 PluginCall (22)-> 179212 PluginCall (22)-> 179212 PluginCall (5)-> 179244 PluginCall (30)-> 179212 PluginCall (22)-> 179212 PluginCall (22)-> 175596 handle_devices-> 178236 sendHeaderBlocking-> 178220 sendContentBlocking-> 176452 sendContentBlocking-> 173100 sendContentBlocking-> 171508 sendContentBlocking-> 166680
> 168985 : Debug dev : 2: lowest: 169924 LoadTaskSettings-> 176920 LoadFromFile-> 176968 sendContentBlocking-> 176796 sendContentBlocking-> 176796 PluginCall (35)-> 171924 PluginCall_task_1 (Unknown)-> 171844 PluginCall (24)-> 175228 PluginCall_task_1 (GET_DEVICEGPIONAMES)-> 175064 sendContentBlocking-> 175108 PluginCall (14)-> 176796 PluginCall_task_1 (Unknown)-> 176724 PluginCall (9)-> 176828 PluginCall_task_1 (WEBFORM_LOAD)-> 176688 sendContentBlocking-> 176652 sendContentBlocking-> 169924
> 168986 : Info   : WD   : Uptime 3 ConnectFailures 0 FreeMem 177424 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 168986 : Debug More : UDP  : Send Sysinfo message
> 169872 : Debug  : LoopStats: shortestLoop: 34 longestLoop: 5818419 avgLoopDuration: 87.00 loopCounterMax: 882352 loopCounterLast: 338908
> 169874 : Debug  : Scheduler stats: (called/tasks/max_length/idle%) 2089352/9459/6/95.19
> 173634 : Debug  : HTML : Request file /connecttest.txt
> 181413 : Debug More : UDP  : 24:6F:28:D1:9A:D8,192.168.3.3,1
> 197297 : Debug  : HTML : Request file /connecttest.txt
> 198983 : Debug dev : Memtrace
>                              0: lowest: 170440 PluginCall_task_1 (Unknown)-> 176724 PluginCall (9)-> 176828 PluginCall_task_1 (WEBFORM_LOAD)-> 176688 sendContentBlocking-> 176652 sendContentBlocking-> 169924 sendContentBlocking-> 176716 sendContentBlocking-> 175068 PluginCall (13)-> 176732 PluginCall_task_1 (Unknown)-> 176660 sendContentBlocking-> 176716 sendContentBlocking-> 175044 handle_devices-> 173268 sendWebPage-> 176620 sendContentBlocking-> 176572 sendContentBlocking-> 170440
> 198985 : Debug dev : 1: lowest: 166680 PluginCall (22)-> 177500 PluginCall (22)-> 179212 PluginCall (22)-> 179212 PluginCall (22)-> 179212 PluginCall (22)-> 179212 PluginCall (5)-> 179244 PluginCall (30)-> 179212 PluginCall (22)-> 179212 PluginCall (22)-> 175596 handle_devices-> 178236 sendHeaderBlocking-> 178220 sendContentBlocking-> 176452 sendContentBlocking-> 173100 sendContentBlocking-> 171508 sendContentBlocking-> 166680
> 198985 : Debug dev : 2: lowest: 169924 LoadTaskSettings-> 176920 LoadFromFile-> 176968 sendContentBlocking-> 176796 sendContentBlocking-> 176796 PluginCall (35)-> 171924 PluginCall_task_1 (Unknown)-> 171844 PluginCall (24)-> 175228 PluginCall_task_1 (GET_DEVICEGPIONAMES)-> 175064 sendContentBlocking-> 175108 PluginCall (14)-> 176796 PluginCall_task_1 (Unknown)-> 176724 PluginCall (9)-> 176828 PluginCall_task_1 (WEBFORM_LOAD)-> 176688 sendContentBlocking-> 176652 sendContentBlocking-> 169924
> 198986 : Info   : WD   : Uptime 3 ConnectFailures 0 FreeMem 178888 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 198986 : Debug More : UDP  : Send Sysinfo message
> 199872 : Debug  : LoopStats: shortestLoop: 34 longestLoop: 5818419 avgLoopDuration: 79.09 loopCounterMax: 882352 loopCounterLast: 371933
> 199874 : Debug  : Scheduler stats: (called/tasks/max_length/idle%) 371933/2131/6/92.05
> 206032 : Debug dev : sendcontent free: 175020 chunk size:1360
> 206044 : Debug dev : sendcontent free: 173436 chunk size:1360
> 206052 : Debug dev : sendcontent free: 166776 chunk size:1360
> 206059 : Debug dev : sendcontent free: 170076 chunk size:1360
> 206075 : Debug dev : sendcontent free: 176540 chunk size:1360
> 206091 : Info   :  Webserver 29 Arguments (First 20): 0: 'index' length: 1 1: 'page' length: 1 2: 'TDNUM' length: 2 3: 'TDN' length: 7 4: 'TDE' length: 2 5: 'serPort' length: 1 6: 'i2cuart_addr' length: 2 7: 'i2cuart_ch' length: 1 8: 'taskdevicepin1' length: 2 9: 'taskdevicepin2' length: 2 10: 'rstpin' length: 2 11: 'pwrpin' length: 2 12: 'p053_model' length: 1 13: 'p053_output' length: 1 14: 'p053_events' length: 1 15: 'p053_delay_wake' length: 2 16: 'p053_oversample' length: 2 17: 'TDT' length: 2 18: 'TDVN1' length: 5 19: 'TDF1' length: 0
> 206100 : Debug dev : Write settings: TaskSettings index: 0
> 206113 : Info   : SaveToFile: free stack: 6792
> 207729 : Info   : FILE : Saved config.dat offset: 32768 size: 536
> 207731 : Info   : SaveToFile: free stack after: 6792
> 207754 : Debug dev : Read settings: TaskSettings index: 0
> 207788 : Debug dev : Write settings: TaskSettings index: 0
> 207791 : Info   : SaveToFile: free stack: 7096
> 209408 : Info   : FILE : Saved config.dat offset: 32768 size: 536
> 209410 : Info   : SaveToFile: free stack after: 7096
> 209411 : Info   : SaveToFile: free stack: 7176
> 211580 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 211581 : Info   : SaveToFile: free stack after: 7176
> 211582 : Debug dev : Write settings: ExtendedControllerCredentials index: 0
> 211600 : Info   : SaveToFile: free stack: 6728
> 211695 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 211697 : Info   : SaveToFile: free stack after: 6728
> 211699 : Debug dev : Read settings: TaskSettings index: 0
> 211736 : Debug  : PMSx003 : config 18 19 -1 -1
> 211737 : Info   : PMSx003 : using hardwaGuru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.
> 
> Core  1 register dump:
> PC      : 0x40093bbe  PS      : 0x00060730  A0      : 0x80094212  A1      : 0x3ffb2180
> A2      : 0xffffffff  A3      : 0x00000017  A4      : 0x3ffc7b1c  A5      : 0x0000abab
> A6      : 0x00060723  A7      : 0x00060720  A8      : 0x00000002  A9      : 0xffffffff
> A10     : 0x00000002  A11     : 0x00000110  A12     : 0x00000017  A13     : 0x0000abab
> A14     : 0x00060723  A15     : 0x00060723  SAR     : 0x00000003  EXCCAUSE: 0x0000001c
> EXCVADDR: 0x0000001b  LBEG    : 0x4008c43d  LEND    : 0x4008c44d  LCOUNT  : 0xffffffff
> 
> 
> Backtrace:0x40093bbb:0x3ffb21800x4009420f:0x3ffb21a0 0x4009447c:0x3ffb21c0 0x4008499a:0x3ffb21e0 0x400849ad:0x3ffb2200 0x40084bc8:0x3ffb2220 0x401f20b9:0x3ffb2240 0x401ec29a:0x3ffb2280 0x401e83ea:0x3ffb22e0 0x400e221a:0x3ffb2320 0x401c1bf7:0x3ffb2350 0x40137ebb:0x3ffb23b0 0x40194008:0x3ffb24c0 0x401e5123:0x3ffb25b0 0x40275151:0x3ffb2660 0x400d80c6:0x3ffb2680 0x400dcc9d:0x3ffb26a0 0x400dcd4d:0x3ffb26e0 0x400dce55:0x3ffb2740 0x4018d8ba:0x3ffb2790 0x4018db71:0x3ffb27c0 0x40125fb3:0x3ffb2800 0x401ec5b4:0x3ffb2820
> 
> 
> 
> 
> ELF file SHA256: 0000000000000000
> 
> Rebooting...
> ets Jul 29 2019 12:21:46
> 
> rst:0xc (SW_CPU_RESET),boot:0x17 (SPI_FAST_FLASH_BOOT)
> configsip: 0, SPIWP:0xee
> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
> mode:DOUT, clock div:2
> load:0x3fff0030,len:184
> load:0x40078000,len:12616
> load:0x40080400,len:2916
> entry 0x400805c4
> ▒U47 : Info   :
> 
> INIT : Booting version: HEAD_3a9783d (ESP32 SDK v4.4.3)
> 48 : Info   : INIT : Free RAM:267164
> 50 : Info   : INIT : Soft Reboot #2 Last Action before Reboot: Background Task Last systime: 1660751447 - Restart Reason: CPU0: Software reset CPU CPU1: Software reset CPU
> 51 : Info   : FS   : Mounting...
> 97 : Info   : FS   : Mount successful, used 151552 bytes of 8318976
> 175 : Info   : CRC  : SecuritySettings CRC   ...OK
> 196 : Info   : INIT : I2C
> 198 : Info   : INIT : SPI not enabled
> 210 : Info   : Time set to 1660751447.000
> 211 : Info   : Current Time Zone: STD time start: 2022-10-30 03:00:00 offset: 360 min
> 213 : Info   : Local time: 2022-08-17 21:50:47
> 226 : Info   : WiFi : Start network scan all channels
> 328 : Info   : WiFi : STA Started
> 7344 : Info   : WiFi : Scan finished, found: 14
> 7347 : Info   : WiFi : Best AP candidate: access_point8 88:DC:96:26:7F:0A Ch:9 (-57dBm) WPA2/PSK
> 7349 : Info   : WIFI : Connecting access_point8 88:DC:96:26:7F:0A Ch:9 (-57dBm) WPA2/PSK attempt #0
> 7359 : Info   : WIFI : Arduino wifi status: WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 7383 : Info   : INIT : Free RAM:218844
> 7407 : Info   : PMSx003 : using hardware serialGuru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.
> 
> Core  1 register dump:
> PC      : 0x40093bbe  PS      : 0x00060f30  A0      : 0x80094212  A1      : 0x3ffb22e0
> A2      : 0xffffffff  A3      : 0x00000017  A4      : 0x3ffc7b1c  A5      : 0x0000abab
> A6      : 0x00060f23  A7      : 0x00060f20  A8      : 0x00000002  A9      : 0xffffffff
> A10     : 0x00000002  A11     : 0x00000110  A12     : 0x00000017  A13     : 0x0000abab
> A14     : 0x00060f23  A15     : 0x00060f23  SAR     : 0x00000003  EXCCAUSE: 0x0000001c
> EXCVADDR: 0x0000001b  LBEG    : 0x4008c43d  LEND    : 0x4008c44d  LCOUNT  : 0xffffffff
> 
> 
> Backtrace:0x40093bbb:0x3ffb22e00x4009420f:0x3ffb2300 0x4009447c:0x3ffb2320 0x4008499a:0x3ffb2340 0x400849ad:0x3ffb2360 0x40084bc8:0x3ffb2380 0x401f20b9:0x3ffb23a0 0x401ec29a:0x3ffb23e0 0x401e83ea:0x3ffb2440 0x400e221a:0x3ffb2480 0x401c1bf7:0x3ffb24b0 0x40137ebb:0x3ffb2510 0x401937db:0x3ffb2620 0x40193e0a:0x3ffb2670 0x401a14e6:0x3ffb2760 0x4018e015:0x3ffb27a0 0x40125fab:0x3ffb2800 0x401ec5a2:0x3ffb2820
> 
> 
> 
> 
> ELF file SHA256: 0000000000000000
> 
> Rebooting...
> ets Jul 29 2019 12:21:46
> 
> rst:0xc (SW_CPU_RESET),boot:0x17 (SPI_FAST_FLASH_BOOT)
> configsip: 0, SPIWP:0xee
> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
> mode:DOUT, clock div:2
> load:0x3fff0030,len:184
> load:0x40078000,len:12616
> load:0x40080400,len:2916
> entry 0x400805c4
> ▒U47 : Info   :
> 
> INIT : Booting version: HEAD_3a9783d (ESP32 SDK v4.4.3)
> 48 : Info   : INIT : Free RAM:267164
> 50 : Info   : INIT : Soft Reboot #3 Last Action before Reboot: Background Task Last systime: 1660751447 - Restart Reason: CPU0: Software reset CPU CPU1: Software reset CPU
> 51 : Info   : FS   : Mounting...
> 97 : Info   : FS   : Mount successful, used 151552 bytes of 8318976
> 175 : Info   : CRC  : SecuritySettings CRC   ...OK
> 196 : Info   : INIT : I2C
> 198 : Info   : INIT : SPI not enabled
> 210 : Info   : Time set to 1660751447.000
> 211 : Info   : Current Time Zone: STD time start: 2022-10-30 03:00:00 offset: 360 min
> 213 : Info   : Local time: 2022-08-17 21:50:47
> 226 : Info   : WiFi : Start network scan all channels
> 328 : Info   : WiFi : STA Started

--- a/platformio_core_defs.ini
+++ b/platformio_core_defs.ini
@@ -218,8 +218,16 @@ platform_packages =
 ; For MUSTFIX_CLIENT_TIMEOUT_IN_SECONDS See: https://github.com/espressif/arduino-esp32/pull/6676
 [core_esp32_IDF4_4__2_0_4]
 ;platform                    = https://github.com/tasmota/platform-espressif32/releases/download/v2.0.4.1/platform-espressif32-2.0.4.1.zip
-platform                    = https://github.com/tasmota/platform-espressif32.git
-platform_packages           =
+
+; debug boot log enabled
+; See: https://github.com/letscontrolit/ESPEasy/pull/4200#issuecomment-1216929859
+;platform                    = https://github.com/Jason2866/platform-espressif32.git
+;platform_packages           = framework-arduinoespressif32 @ https://github.com/Jason2866/esp32-arduino-lib-builder/releases/download/936/framework-arduinoespressif32-443_esp421-9ce849ce72.tar.gz
+
+; debug boot log disabled
+platform                    = https://github.com/Jason2866/platform-espressif32.git
+platform_packages           = framework-arduinoespressif32 @ https://github.com/Jason2866/esp32-arduino-lib-builder/releases/download/938/framework-arduinoespressif32-443_esp421-10ab11e815.tar.gz
+
 build_flags                 = -DESP32_STAGE
                               -DMUSTFIX_CLIENT_TIMEOUT_IN_SECONDS
                               -DLIBRARIES_NO_LOG=1

--- a/src/src/PluginStructs/P053_data_struct.cpp
+++ b/src/src/PluginStructs/P053_data_struct.cpp
@@ -258,7 +258,7 @@ bool P053_data_struct::processData(struct EventStruct *event) {
   }
   uint16_t data[PMS_RECEIVE_BUFFER_SIZE] = { 0 }; // uint8_t data_low, data_high;
 
-  for (uint8_t i = 0; i < frameData; i++) {
+  for (uint8_t i = 0; i < frameData && i < PMS_RECEIVE_BUFFER_SIZE; i++) {
     SerialRead16(data[i], &checksum);
   }
 

